samples/Sentry.Samples.Console.Basic/runtimeconfig.template.json

```diff
@@ -0,0 +1,6 @@
+{
+  "configProperties": {
```

Comment:
do we need this file if there's a new sample?

Reply:
We need it in the new sample actually... I've moved it across - tnks!